fdbrpc/AsyncFileEncrypted.actor.cpp

```diff
@@ -196,7 +203,7 @@ int64_t AsyncFileEncrypted::debugFD() const {
 	return file->debugFD();
 }
 
-StreamCipher::IV AsyncFileEncrypted::getIV(uint16_t block) const {
+StreamCipher::IV AsyncFileEncrypted::getIV(uint32_t block) const {
```

Comment:
Block `x` and block `(x + (1<<16))` will now have the same IV, so we should fix this.